lib/src/data/communication/api/api_errors.dart

```diff
@@ -0,0 +1,33 @@
+abstract class ApiError {
+  final int _statusCode;
+  final String _message;
+
+  ApiError(this._message, [this._statusCode]);
+
+  @override
+  String toString() => 'message: $_message,statusCode: $_statusCode';
+}
+
+class BadRequestError extends ApiError {
+  BadRequestError(statusCode, [message = "Bad request error"]) : super(message, statusCode);
```

Comment:
W jakim celu message?

Reply:
idea była taka, żeby to wyświetlić na UI

Reply:
jako, że obsługujemy dwa języki to nie zadziała

Reply:
dodam do tłumaczeń

Reply:
tak, tylko wtedy będzie to to bazować na warstwie UI, bo wyciągasz na podstawie contextu, więc proponowałbym wywalić message z tych errorów i zrobić coś ala exceptionHandler, który na podstawie domenowego exceptionu by zwracał komunikaty błędu, wtedy byłoby to w warstwie prezentacji i wszystko cacy :)